unicorn.rb

```diff
@@ -0,0 +1,23 @@
+class Unicorn
+  attr_accessor :name
```

Comment:
I think this could be an attr_reader - is there a reason that we are exposing this?